### ISSUE TITLE
freePort("auto") doesn't work if R_PARALLEL_PORT is not NULL.

### ISSUE THREAD

**Issue by randy3k** [HenrikBengtsson/parallelly#57]
```
Sys.setenv(R_PARALLEL_PORT = 8888)
library(future)
plan(multisession)
> Error in freePort(port) : Unknown value on argument 'port': 'auto'
```

**Comment by HenrikBengtsson** [comment]
Thank you. A minimal example is:

```r
> Sys.setenv(R_PARALLEL_PORT = 8888)
> cl <- parallelly::makeClusterPSOCK(1L)
Error in freePort(port) : Unknown value on argument 'port': 'auto'
```

**Comment by HenrikBengtsson** [comment]
Fixed in parallelly (>= 1.26.1-9001), cf. commit ac1f5c7

```r
> Sys.setenv(R_PARALLEL_PORT = 8888)
> cl <- parallelly::makeClusterPSOCK(1L)
TRACKER: loadedNamespaces() changed: 2 packages loaded ('parallel', 'parallelly')
> cl[[1]]$con
A connection with                              
description "<-localhost:8888"
class       "sockconn"        
mode        "a+b"             
text        "binary"          
opened      "opened"          
can read    "yes"             
can write   "yes" 
```
